app/templates/talks/talkBranded/page.tsx

```diff
@@ -95,6 +94,82 @@ export default function BrandedTalkPage() {
 		speakersJob,
 	});
 
+	const formConfig: formConfigProps = {
+		backgroundColor: {
+			state: backgroundColor,
+			setState: setBackgroundColor,
+			label: 'Background Color (optional)',
+			component: ColorInput,
```

Comment:
Ça sera forcément des inputs non ?
```suggestion
			inputType: ColorInput,
```

Reply:
InputComponent ? inputType avec TS je trouve que ça prête à confusion.